data-access-analysis-tools/index-nb-intro.rst

```diff
@@ -0,0 +1,141 @@
+Return to this launch pad at any time by clicking the plus symbol at upper left.
+For DP0.1, only Python Notebooks and the terminal interface are supported (i.e., not the Console), and users are not able to access Portal queries from a Jupyter Notebook.
+
+.. figure:: /_static/RSP_NB_launcher_options.png
+    :width: 400
+    :name: RSP_NB_launcher_options
+
+    The JupyterLab landing page from which several tools can be launched and the file system can be browsed (left sidebar).
+
+In the very left-most vertical sidebar of icons, the top icon is a file folder, and that is the default view.
+The left sidebar lists folders in the user's home directory (e.g., DATA, WORK, and notebooks).
+Launching a terminal (the default is a linux bash terminal) and using the command "ls" will return the same list.
+Navigate the file system and open files by double-clicking on folders and files in the left sidebar.
+All users will find a set of tutorial notebooks provided in the notebooks directory (:ref:`Examples-DP0-1-Notebooks`).
+Jupyter Notebooks can be identified by their suffix ".ipynb". 
+
+**Using a Terminal:**
+The DP0.1 data sets and the LSST Science Pipelines tools can both be accessed from the command line of a terminal. As described in the default message that appears in all newly-launched terminals, to create a Rubin Observatory environment in a terminal session and set up the full set of packages, users must first execute "source ${LOADSTACK}" and "setup lsst_distrib". For example, to query and retrieve DP0.1 data sets using the Butler (see below), command-line tools are available as documented `here <https://pipelines.lsst.io/v/weekly/modules/lsst.daf.butler/scripts/butler.html>`_. Type "butler --help" in any terminal to see a list of available butler functionality.
+
+**Safely Log Out of JupyterLab:**
+Use the File item in the top menu bar.
+To safely shut down a Notebook, choose Close and Shutdown Notebook.
+To safely shut down a JupyterLab server and log out of the RSP, choose Save all, Exit, and Log Out.
+
+
+.. _NB-Intro-Use-A-NB:
+
+How to Use a Jupyter Notebook
+=============================
+
+The best way to learn how to use a Jupyter Notebook is to open the first of the tutorial notebooks which are provided in all users' home directories, and also available in the `tutorial-notebooks <https://github.com/rubin-dp0/tutorial-notebooks>`_ repository in the ``rubin-dp0`` GitHub Organization (see also :ref:`Examples-DP0-1-Notebooks`). 
+
+**Executing code in a Notebook:**
+Jupyter notebooks provide "cells" within which you type either Python code or markdown language (for formatted text).
+Choose the cell to execute by clicking in it with your mouse (the cursor must be in the desired cell).
+Hold down the `shift` key and press either `enter` or `return` (depending on your keyboard type), and the contents of the cell will be executed.
+If the cell type is code, and the cell contains python code, the code will be compiled and executed.
```

Comment:
Small point, but python is an interpreted language.  Since there really isn't a compilation step, I think you can just say "the code will be executed"